fidouafclient/app/src/main/java/org/ebayopensource/fidouafclient/op/Reg.java

```diff
@@ -38,17 +38,22 @@
 public class Reg {
 	
 	private Gson gson = new GsonBuilder().disableHtmlEscaping().create();
-	
+
 	public String getUafMsgRegRequest (String username, String appId){
```

Comment:
Hi Neb,
I wrote a new fix to this issue according to the algorithms in [FIDOAppIDAndFacets](https://fidoalliance.org/specs/fido-uaf-v1.0-ps-20141208/fido-appid-and-facets-v1.0-ps-20141208.html#the-appid-and-facetid-assertions). Please, have a look on this new pull request.

Now I'm  quite sure that is necessary to change [getAppId method in FidoUafResource class - ](https://github.com/eBay/UAF/blob/master/fidouaf/src/main/java/org/ebayopensource/fidouaf/res/FidoUafResource.java#L149) and [facets method in FidoUafResource class - FIDO Server](https://github.com/eBay/UAF/blob/master/fidouaf/src/main/java/org/ebayopensource/fidouaf/res/FidoUafResource.java#L132) because they are defining statically trustedFacets and appId as `https://www.head2toes.org/fidouaf/v1/public/uaf/facets`. People who is using your code should use their own fido server instance and put statically their android application hash in `facets` method.